### PR TITLE
02_pytorch_classification.ipynb variable name fix

### PR DESCRIPTION
**Pull request by snsa-kscc**
Hey Daniel!
Great work. Excellent teaching style. Grateful immensely! 

Just before applying softmax on logits (multi-class PyTorch model) there is an error:

`y_logits = model_4(X_test.to(device))`

Correct:

`y_logits = model_4(X_blob_test.to(device))`

I have included it in PR.

Cheers!

### CODE REVIEW

**Comment by mrdbourke** [comment]
thank you so much! I really appreciate it!
